delete_mpx.go

```diff
@@ -0,0 +1,172 @@
+package dbr
+
+import (
+	"context"
+	"database/sql"
+	"strconv"
+)
+
+// DeleteStmtMpx builds `DELETE ...`.
```

Comment:
Is there a different factoring of these changes where only the `exec()` or `query()` callback that the DeleteStmt, for example, uses gets factored out? Something like:

```
type DeleteStmt struct {
        EventReceiver
        Dialect

        raw

        Table      string
        WhereCond  []Builder
        LimitCount int64

        comments Comments

        exec func(context.Context, builder Builder) (sql.Result, error)
}
```

And then when you construct a non-multiplexed DeleteStmt, you give it something like:

```
func(ctx context.Context, b Builder) (sql.Result, error) {
    return unaryExec(ctx, sess, sess.EventReceiver, b, sess.Dialect)
}
```

and when you construct a multiplexed DeleteStmt, you give it something like:

```
func(ctx context.Context, b Builder) (sql.Result, error) {
    return multiExec(ctx, sess.primaryRunner, sess.secondaryRunner, sess.EventReceiver, sess.secondaryRunner, b, sess.Dialect, sess.secondaryDialect)
}
```

or whatever?

Basically, can you structure it to switch on that one call in ExecContext/LoadContext?